### ISSUE TITLE
Export / PDF Generation

### ISSUE THREAD

**Issue by Frencil** [statgen/locuszoom#22]
A potential use case for LocusZoom in the future is means of exporting a LocusZoom instance to a graphic or PDF suitable for inclusion in a paper.

This feature is arguably out of scope of the plugin itself and will require a companion repository for server-side installation.

## Estimate

This feature will probably take at least a month of development to build and test because it branches out into a pretty different technology stack from the plugin itself.

**Comment by Frencil** [comment]
**Update:** A proof of concept of this is now on the [export](https://github.com/statgen/locuszoom/tree/export) branch. It turns out this does *not* need to be a separate server-side module.

As demonstrated in [ExAC](http://exac.broadinstitute.org/) one stream an SVG element into a Base64-encoded string and then into a link for download as a data stream entirely on the client side.

This task will still take some work to be a fully-functional feature in the plugin itself, but it can definitely be done and will be simpler in scope.

**Comment by Frencil** [comment]
**Update:** This is now addressed in #27 but is blocked by #30. That PR establishes a new syntax for serializable layouts which will allow for more intuitive optional inclusion of the controls element (and developer customization thereof) by refactoring this feature as an optional "HTML module".

The same pattern is being proposed for #21, which also has #30 as a blocker.

**Comment by Frencil** [comment]
Released in v0.3.9.
